packages/neuron-wallet/tests/block-sync-render/queue.test.ts

```diff
@@ -170,10 +170,17 @@ describe('queue', () => {
       await queue.start()
     });
     it('inits IndexerConnector', () => {
+      let expectPathRegex
+      if (process.platform === 'win32') {
+        expectPathRegex = expect.stringMatching(/test\\indexer_data/)
```

Comment:
The `\\` and `/` could be replaced by `path.delimiter`

Reply:
Not sure how to use `path.delimiter` in ragex here. Any suggestions?

Reply:
`new RegExp('text' + path.delimiter + 'index_data')` should work

Reply:
my fault, use `path.sep`

Reply:
Thanks! It works.

Reply:
Actually the[ test failed ](https://dev.azure.com/nervosnetwork/neuron/_build/results?buildId=12734&view=logs&jobId=1dd1a9fa-5bd6-5a58-be75-4682264ffaf4&j=1dd1a9fa-5bd6-5a58-be75-4682264ffaf4&t=0fd07730-01dd-5646-2168-32456ff1e2aa)on window when using `path.sep`, which can't automatically make the path separator compatible on windows.

I revert the code for this test.